__tests__/cron/cleanGiftedPlus.ts

```diff
@@ -0,0 +1,70 @@
+import cron from '../../src/cron/cleanGiftedPlus';
+import { expectSuccessfulCron, saveFixtures } from '../helpers';
+import { User } from '../../src/entity';
+import { plusUsersFixture, usersFixture } from '../fixture';
+import { DataSource, JsonContains } from 'typeorm';
+import createOrGetConnection from '../../src/db';
+import { addDays, subDays } from 'date-fns';
+import { SubscriptionCycles } from '../../src/paddle';
+
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+});
+
+beforeEach(async () => {
+  await saveFixtures(con, User, [...usersFixture, ...plusUsersFixture]);
+});
+
+it('should remove expired gifted plus subscription', async () => {
```

Comment:
can you use `describe` to group it, easier to add multiple tests after?

and also add `should be registered` test (check other crons for example)

Reply:
Updated ✔️